### ISSUE TITLE
Investigate mailchimp syncing issue

### ISSUE THREAD

**Issue by bmpickford** [govhackaustralia/hackerspace3#416]

**Comment by astley92** [comment]
Just wondering what you mean by this

> Mailchimp integration has not been working for the past few years.

Does this mean we haven't been updating the mailing lists or is it that our mailchimp subscription/account/config is off and it's not working at all, even for the existing mailing list?

Happy to have a look at this one if you haven't already started?

If you are already looking, the bulk of mailchimp stuff is done as a background job in `MailchimpUpdateJob` happy to chat if it helps.

**Comment by bmpickford** [comment]
Yeah it still needs some investigation, but essentially our mailing list isn't being updated for some reason. I've started investigating but it's still unclear as to whether it's config or code.

**Comment by astley92** [comment]
Yeh gotcha, thanks for that. Do we use a logging service? Papertrail or anything? That's where I'd start but I don't really have access to anything sorry. Also looks like were just using the default queue adapter for active job, should probably make sure thats working as expected.
